### ISSUE TITLE
VarianceScaling Initializer Is Unseeded

### ISSUE THREAD

**Issue by cn4750** [LeelaChessZero/lc0#1883]
@Tilps ran into this issue after upgrading TensorFlow:
> /usr/local/lib/python3.8/dist-packages/keras/initializers/initializers.py:120: UserWarning: The initializer VarianceScaling is unseeded and being called multiple times, which will return identical values each time (even if the initializer is unseeded). Please update your code to provide a seed to the initializer, or avoid using the same initalizer instance more than once.

From Tilps:
> I think its a bug in our code - I think you are supposed to construct a separate instance for every use case, so that 'call' is only invoked at most once.

I assume it is [this](https://github.com/LeelaChessZero/lczero-training/blob/master/tf/tfprocess.py#L1356-L1357) code the above warning references.

@masterkni6, can you look at this when you get a chance?
